src/utils/people.ts

```diff
@@ -261,3 +270,66 @@ export function determineBadgeHost(badgeCode: number) {
   const badge_host = { 1: 'liquid.sphinx.chat' }
   return badge_host[badgeCode]
 }
+
+export async function reissueBadgeOnLiquid({
+  amount,
+  badge_id,
+  owner_pubkey,
+}: reissueOnLiquid) {
+  try {
+    const token = await genSignedTimestamp(owner_pubkey)
+    const r = await fetch(
+      `${config.boltwall_server}/reissue_badge?token=${token}`,
```

Comment:
I dont think this is added on boltwall yet can we change this to `v1/reissue_badge`